src/DataLib/fendl32B_retrofit/process_fendl3.2.py

```diff
@@ -13,13 +13,20 @@ def main():
     endf_path = 'tape20'
     pendf_path = 'tape21'
 
-    material_id, MTs = tp.extract_endf_specs(endf_path)
+    material_id, MTs, endftk_file_obj = tp.extract_endf_specs(endf_path)
+    MTs = list(set(MTs).intersection(mt_dict.keys()))
```

Comment:
This doesn't need to be a `list` after this set operation.  Forcing it to be a list is not wrong, but it conveys that a list is required.  (I won't hold up the merge for this)